gc/base/standard/Scavenger.cpp

```diff
@@ -1677,6 +1677,12 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 	return destinationObjectPtr;
 }
 
+MMINLINE void
+MM_Scavenger::fixupForwardedObject(MM_ForwardedHeader* forwardedHeader, omrobjectptr_t destinationObjectPtr, uintptr_t objectAge) {
+	/* Object model specific fixup, like age */
+	_extensions->objectModel.fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
+}
+
```

Comment:
@amicic maybe this helper is overkill now? Before we were also updating `dataAddr` here but now we don't need to do that anymore.

Reply:
correct, not needed, anymore